woocommerce/woocommerce-android
# Issue/4973 editable order field

Closes #4973 - This PR implements the "Add customer note" view which should appear when the customer note is empty. The main work in this PR was to create `WCActionableEmptyLabel`. This widget displays an "actionable" view when the field is empty:

![action](https://user-images.githubusercontent.com/3903757/137016630-b69295ad-bef2-41e2-848e-07808b47ba73.png)


Otherwise it displays the field's text with a pencil icon:

![notempty](https://user-images.githubusercontent.com/3903757/137016646-50a39ec1-af14-4492-9f34-1be1c8456d36.png)


This should simplify order editing because we can re-use it for billing address, shipping address, etc.

To test:
* Go to order detail for an order that has a customer note and verify the note appears as expected
* Go to order detail for an order that does NOT have a customer note and verify the "Add customer note" view appears
